### ISSUE TITLE
Remove `IfcSimplePropertyTemplateTypeEnum.Q_NUMBER`

### ISSUE THREAD

**Issue by pjanck** [bSI-InfraRoom/IFC-Specification#683]
See https://github.com/buildingSMART/IFC4.3.x-development/issues/167

**Comment by SergejMuhic** [comment]
Hm, seems as though there is a description for it missing. Also `IfcSimplePropertyTemplate`.`Expression` does not say anything about it either. Should this be amended?

ping @aothms @larswik @Michel-Rives @anborr @TLiebich

**Comment by SergejMuhic** [comment]
To my understanding this should be add Q_NUMBER. But since there was no response, I have created a new issue with the action description: https://github.com/bSI-InfraRoom/IFC-Specification/issues/765

Still waiting on some feedback regarding changes to the description of `IfcSimplePropertyTemplate`.`Expression` and the description of the constant.
